metriq_gym/quantinuum/device.py

```diff
@@ -0,0 +1,146 @@
+from __future__ import annotations
+
+import uuid
+from typing import Any, cast
+
+from qbraid import QPROGRAM, load_program
+from qbraid.programs import ExperimentType, ProgramSpec
+from qbraid.runtime import DeviceStatus, QuantumDevice, TargetProfile
+from qiskit import QuantumCircuit
+
+from .job import QuantinuumJob
+from .auth import load_api
+from metriq_gym.local._store import write
+
+
+SUPPORTED_EMULATORS = {
+    # Quantinuum emulator device identifiers (NEXUS)
+    "H1-1E": {"num_qubits": 20},
+    "H1-2E": {"num_qubits": 20},
+}
```

Comment:
In other qBraid providers, this is not hardcoded, we support all devices. Why do we need this here?

Reply:
You’re right, we shouldn’t hardcode device IDs. Done in 499e41ff3d3784f25445244e95927fbf97754cef